client/packages/common/src/ui/layout/tables/material-react-table/components/ChipTableCell.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Box, Chip } from '@openmsupply-client/common';
+import { MRT_Cell, MRT_RowData } from 'material-react-table';
+
+export const ChipTableCell = <T extends MRT_RowData>({
+  cell,
+}: {
+  cell: MRT_Cell<T>;
+}) => {
+  const items = cell.getValue<string[]>();
+  return (
+    <Box
+      sx={{
+        flexDirection: 'row',
+        borderBottom: 'none',
+        alignItems: 'center',
+        display: 'flex',
+        flexWrap: 'wrap',
+        gap: 0.2,
+        padding: '3px 0',
```

Comment:
Can remove `borderBottom`, `alignItems` and `padding`. The first two are redundant, and the padding one is adding too much padding -- the table already has enough, and with this one on the row actually ends up being taller in "compact" view than in the default view.

(Cell components shouldn't really add their own padding, the table should handle it, and we can add some to specific column definitions if we really need it -- such as the left padding for the "Code" column)